Lucheti29/Training
# Create news and News detail implemented

## Summary

'News detail' and 'Create news' frontend implemented.
## Test
- Login with a Parse user account (for example, username: t@w.com // password: asdf)
- Change to News tab and click a new to see the detail
- On detail layout, click on 'Back arrow' to go back
- Click on Fab button to open 'Create news' view
## Screenshot

![News detail](http://i19.servimg.com/u/f19/12/28/53/00/news_d11.jpg)

![Create news](http://i19.servimg.com/u/f19/12/28/53/00/create10.png)
## Cards

https://trello.com/c/3u8rb0Ts/18-news-detail
https://trello.com/c/Qt18sjQb/19-create-news
